plugins/mcreator-core/datalists/blocksitems.yaml

```diff
@@ -3710,12 +3709,6 @@
   type: block
   texture: CONCRETE_POWDER#15
 
-- Blocks.CONCRETE:

```

Comment:
The changes for MC's tags are fine (and the FB Generator thanks you!). However, concerning the 2 removed entries... not too sure it's a good idea. See the image
![image](https://github.com/MCreator/MCreator/assets/38427877/0e8ed9ac-162d-4873-8f61-a7b08a6dac0a)

The missing entries are converted to air... it doesn't break workspaces, but I can already some potential issues:
1. My ReCiPeS (or whatever it is) ArE WeIrD! HELP ME PLEASE!!!
2. This might break break some custom logic (mainly in procedures I think) made by users.

However, as they weren't real wildcards, this might be easily fix by automatically change them to White concrete (powder), so users don't notice.

Reply:
I thought to just remove them because they don't work the way users would want them to, so even if we convert them to white concrete, it won't work as "any concrete" as the user may anticipate.

The only place where they could use them so far was in recipes because it had "subtypes" and thus did not load when tags were not allowed